public/v4/apps/weblate.yml

```diff
@@ -0,0 +1,88 @@
+captainVersion: 4
+services:
+    $$cap_appname-redis:
+        image: redis:$$cap_redis_version
+        volumes:
+            - $$cap_appname-cache-data:/data
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname-postgres:
+        image: postgres:$$cap_postgres_version
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+        restart: always
+        environment:
+            POSTGRES_USER: weblate
+            POSTGRES_PASSWORD: $$cap_pg_pass
+            POSTGRES_DB: weblate
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: weblate/weblate
+        depends_on:
+            - $$cap_appname-postgres
+            - $$cap_appname-redis
+        volumes:
+            - $$cap_appname-data:/app/data
+        restart: always
+        environment:
+            DEBUG: 0
+            WEBLATE_LOGLEVEL: INFO
+            WEBLATE_ADMIN_EMAIL: $$cap_admin_email
+            WEBLATE_ADMIN_PASSWORD: $$cap_admin_password
+            WEBLATE_SERVER_EMAIL: $$cap_server_email
+            WEBLATE_DEFAULT_FROM_EMAIL: $$cap_email_from
+            WEBLATE_REGISTRATION_OPEN: $$cap_open_registration
+            POSTGRES_PASSWORD: $$cap_pg_pass
+            POSTGRES_USER: weblate
+            POSTGRES_DATABASE: weblate
+            POSTGRES_HOST: srv-captain--$$cap_appname-postgres
+            REDIS_HOST: srv-captain--$$cap_appname-redis
+            WEBLATE_SITE_DOMAIN: $$cap_appname.$$cap_root_domain
+            WEBLATE_SITE_TITLE: $$cap_appname Weblate
+            WEBLATE_ADMIN_NAME: Weblate Admin
+            POSTGRES_PORT: 5432
+            REDIS_PORT: 6379
+        caproverExtra:
+            containerHttpPort: '8080'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_postgres_version
+          label: Postgres Docker version
+          defaultValue: 13-alpine
+          description: Check out their Docker page for the valid versions https://hub.docker.com/r/library/postgres
+        - id: $$cap_redis_version
+          label: Redis Docker version
+          defaultValue: 6-alpine
+          description: Check out their Docker page for the valid versions https://hub.docker.com/r/library/redis
+        - id: $$cap_weblate_version
+          label: weblate Docker Image SHA Hash
```

Comment:
Based on the label
if we need to use hash instead of tag
you need use image: `weblate/weblate@$$cap_weblate_version` at line 21

Reply:
Generally, if a proper tag is available, it should be the preferred option as it's much more readable than the hash. The only benefit of hash is that it's not mutable.